packer.mk

```diff
@@ -5,10 +5,7 @@ ifeq (${OS_SKU},Ubuntu)
 ifeq (${ARCHITECTURE},ARM64)
 ifeq (${MODE},gen2Mode)
 	@echo "${MODE}: Building with Hyper-v generation 2 ARM64 VM"
-	@rm -f ./vhdbuilder/packer/packer
-	@curl -fsSL https://packerbinary.blob.core.windows.net/bin/packer -o ./vhdbuilder/packer/packer
```

Comment:
@HDYA  JFYI. In an AKS pipeline, you also use my private packer. Now you can use packer 1.8.1+ with 'skip_create_image' setting to workaround an ARM64 stack restriction.